_data/categories/meshes.yml

```diff
@@ -24,6 +24,19 @@
   tracing: "Pluggable"
   encryption: "Yes"
 
+- name: Grey Matter
+  desc: "Grey Matter is an enterprise framework and ecosystem covering common use cases and patterns, so your team can focus on what they do best."
```

Comment:
@joshua-rutherford @brianlitalien it's good to have Grey Matter represented. Thanks for contributing. Can you take another crack at the description, refactoring the latter portion (", so your team can focus on what they do best.") to be more objective in nature (e.g. "built on top of xxx technology" or "deployable with or without a client library" or "deployable with or without a service proxy")?